packages/editor-ui/src/components/MainHeader/WorkflowDetails.vue

```diff
@@ -141,6 +141,7 @@ export default mixins(workflowHelpers).extend({
 	},
 	methods: {
 		onSaveButtonClick () {
+			this.$store.dispatch('settings/getPromptsData');
```

Comment:
IMPORTANT - we should also wrap every dispatch that makes an api request in try and catch... in this case, if the api request fails, it will stop saving.. 
n8n should still work if you are running server locally and you are offline... in this case, our api can block saving

Reply:
Now we have here this.$store.dispatch('ui/openUserPromptsIfPossible'); just to prevent the unnecessary request to the server. It is placed after saving the workflow now. And I wrapped all requests with try catch. :)